themes/he/layouts/partials/collect-info-modal.html

```diff
@@ -83,9 +83,12 @@
                       type="number"
                       name="phone_number"
                       placeholder="Phone number*"
+                      
```

Comment:
check and remove this space. @arunkumar295

Reply:
white space removed as you said.